### ISSUE TITLE
Mined a block but no reward received

### ISSUE THREAD

**Issue by appkoders** [dinastyoffreedom/dinastycoin#4]
I started  my own pool and I succesfully mined blocks. But the first 3 blocks I mined did not give me any reward.  The rest of the blocks I mined after the first 3 blocks had rewards. 

Here is my pool: https://dcy.cryptomineros.com/#pool_blocks

The blocks from block height 351432, 351430 and 351331 were the blocks that did not give me the rewards.

What could be the reason why I did not receive the rewards? I am sure they will go somewhere else so  is there any way to check on what address was the rewards sent?

**Comment by dinastyoffreedom** [comment]
check out the config.json there the address it is set. and then are you runnng the simplewalelt in RPC mode?

**Comment by appkoders** [comment]
when I first started the pool the config is already set. Everything is fine except that the first 3 blocks has no reward and the rest is just fine and I did not modify the config after the first 3 blocks so I guess the config.json is correct. I believe the simplewallet is not running on the first few hours when I started it since I just thought I will just start the wallet after I mined a block. Will I get no reward if the simplewallet is not running? I thought the simplewallet is only used to send payouts or it is needed after you mine a block?
